src/pretix/base/services/placeholders.py

```diff
@@ -337,6 +337,39 @@ def base_placeholders(sender, **kwargs):
                 }
             ),
         ),
+        SimpleFunctionalTextPlaceholder(
+            'url_info_change', ['position', 'event'], lambda position, event: build_absolute_uri(
+                event,
+                'presale:event.order.position.modify', kwargs={
+                    'order': position.order.code,
+                    'secret': position.web_secret,
+                    'position': position.positionid
+                }
+            ), lambda event: build_absolute_uri(
+                event,
+                'presale:event.order.position.modify', kwargs={
+                    'order': 'F8VVL',
+                    'secret': '6zzjnumtsx136ddy',
+                }
```

Comment:
...sonst crasht die mail-configuration
```suggestion
                    'secret': '6zzjnumtsx136ddy',
                    'position': '123',
                }
```